Makefile

```diff
@@ -438,7 +439,7 @@ up-detach: build ## Bring up local Grapl and detach to return control to tty
 	unset COMPOSE_FILE
 	docker-compose \
 		--file docker-compose.yml \
-		up --detach --force-recreate --always-recreate-deps --renew-anon-volumes pulumi
+		run pulumi
```

Comment:
tbh I can't guarantee that `run` will have the same properties, with renew anon volumes etc

i foudn the following "works"
```

	# TODO: This could potentially be replaced with a docker-compose run, but
	#  it doesn't have all these useful flags
	docker-compose \
		--file docker-compose.yml \
		up --force-recreate --always-recreate-deps --renew-anon-volumes \
		--exit-code-from pulumi \
		pulumi
```